models/library.go

```diff
@@ -146,7 +146,7 @@ var FindLockFiles = []string{
 	// gomod
 	ftypes.GoMod, ftypes.GoSum,
 	// java
-	ftypes.MavenPom, "*.jar", "*.war", "*.ear", "*.par",
+	ftypes.MavenPom, "*.jar", "*.war", "*.ear", "*.par", "gradle.lockfile",
```

Comment:
According to the Trivy documentation, `*gradle.lockfile` is targeted.
https://aquasecurity.github.io/trivy/v0.35/docs/vulnerability/detection/language/

```suggestion
	ftypes.MavenPom, "*.jar", "*.war", "*.ear", "*.par", "*gradle.lockfile",
```